apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoardTest.js

```diff
@@ -452,6 +452,25 @@ describe('CircuitPlaygroundBoard', () => {
     });
   });
 
+  describe(`reset()`, () => {
+    it('stops any created Leds', () => {
```

Comment:
Giving that we are now turning the component off, should this test a call to the led.off?

Reply:
`.off()` internally calls `.stop()` [here](https://github.com/code-dot-org/code-dot-org/blob/68d68cf998d13ebbe01460750994d681dcb85c29/apps/src/lib/kits/maker/boards/circuitPlayground/Led.js#L11), so I think it's okay to keep this test as-is.  It essentially replicates the test for `destroy()`, [below](https://github.com/code-dot-org/code-dot-org/pull/45945/files#diff-d59eea7e2eb1c3dd6fcb8c232d712e2059d66a2855ba0c16e9ddece73b3a9227R516-R517).

Reply:
I'm tempted to add a short comment here that captures that^, so that it's easy to read the original file and test against each other and see the parallel without diffing in to implementation. Definitely a nit and no need to hold merge, but just a thought.